packages/editor-ui/src/components/NodeCredentials.vue

```diff
@@ -101,15 +101,19 @@ export default mixins(
 		}),
 		isProxyAuth(): boolean {
 			return this.isHttpRequestNodeV2(this.node) &&
-				this.node.parameters.authenticateWith === 'nodeCredential';
+				this.node.parameters.authentication === 'existingCredentialType';
 		},
 		isGenericAuth(): boolean {
 			return this.isHttpRequestNodeV2(this.node) &&
-				this.node.parameters.authenticateWith === 'genericAuth';
+				this.node.parameters.authentication === 'genericCredentialType';
 		},
 		credentialTypesNode (): string[] {
-			return this.credentialTypesNodeDescription
+			const nodeCredentialTypes = this.credentialTypesNodeDescription
 				.map((credentialTypeDescription) => credentialTypeDescription.name);
+
+			this.$emit('nodeCredentialTypes', nodeCredentialTypes);
```

Comment:
getters should not have side effects... can put this in a watch.. or better duplicate logic in parent.